operator/pkg/model/translation/gateway-api/translator_test.go

```diff
@@ -35,6 +35,7 @@ func Test_translator_Translate(t *testing.T) {
 		{name: "basic_http_listener_nodeport"},
 		{name: "basic_http_listener_external_traffic_policy"},
 		{name: "basic_http_listener_load_balancer"},
+		{name: "basic_http_listener_clusterip"},
```

Comment:
seems like this is the only change, and the test is passing :thinking:. 

Do we need to change anything in translator.go ?